docs/api/exporter.rst

```diff
@@ -0,0 +1,13 @@
+.. py:module:: vortex.development.exporter.onnx
+.. py:currentmodule:: vortex.development.exporter.onnx
+
+Exporter
+========
+
+Vortex includes onnx exporter to ease exporting model and embedding model properties.
+
+.. autoclass:: vortex.development.exporter.onnx.ONNXExporter
+  :members:
+
+  .. automethod:: __init__
+  .. automethod:: __call__
```

Comment:
![image](https://user-images.githubusercontent.com/18413229/113539459-0ef87200-9608-11eb-96a9-4af90245f974.png)

it's a little weid to see the second part first, could you change it to the correct order?

Reply:
hm let me see, because the py script is already in order, so i think this is autodoc's thing

Reply:
yepp, I think you need to call each function one by one with the `.. automethod`

Reply:
updated